Containerfile

```diff
@@ -0,0 +1,51 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3.11
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+# Copy the files of esi leap inside the container
+# Syntax : COPY <src> <destination>
+
+COPY run_services.sh run_services.sh
+
+# Please make sure to have esi-leap.conf file in same directory as containerfile 
+# or make sure to mention the right path of the file below.
+
+#COPY esi-leap.conf /etc/esi-leap/esi-leap.conf
```

Comment:
These lines should be removed from the Containerfile since they no longer apply to how we are currently doing things